### ISSUE TITLE
[DRAFT] Long-term solution to large file storage 

### ISSUE THREAD

**Issue by eddietejeda** [18F/federalist#1995]
#### User Story

A user of Federalist who wants to upload large files is blocked by limits in container size. 

More information:
https://gsa-tts.slack.com/archives/C04823NMA/p1487884812002420

#### Acceptance Criteria

- [ ]  Users are able to have large containers

#### Implementation outline (if higher than "low" effort):

- [ ] Investigate Git LFS
- [ ] Investigate whitelisting files or directories
other options?

**Comment by eddietejeda** [comment]
Follow up with ONNR to discuss their build process in more detail.

**Comment by eddietejeda** [comment]
This issue is going into icebox for a while. We're about to make changes to our infrastructure and need to determine where this fits in.

**Comment by eddietejeda** [comment]
Can we evaluate this: https://aws.amazon.com/athena/

**Comment by davemcorwin** [comment]
1. @eddietejeda you may want to create a new issue for looking into Athena. This issue looks to be about large static resources, such as images and pdfs.
2. It might be useful to come up with a use case for this. It looks like its focus is reading large datasets that already exist in S3, such as logging, so would our users want a feature to upload json files of data to be queried?
